root/artist/header.tt

```diff
@@ -1,10 +1,11 @@
-<div class="artistheader">
+[% header = 'artistheader ' _ (html_escape(artist.l_type_name) or l('Artist')) _ 'ico' %]
```

Comment:
Can we append `-icon` instead of `ico`? It's easier to read and there's no reason to abbreviate it.